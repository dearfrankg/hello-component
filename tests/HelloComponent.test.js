import React from "react";
import HelloComponent from "../src";

describe("Hello Component", () => {
  it("should render", () => {
    const wrapper = shallow(<HelloComponent />);
    const actual = wrapper.exists();
    expect(actual).toMatchSnapshot();
  });
});
