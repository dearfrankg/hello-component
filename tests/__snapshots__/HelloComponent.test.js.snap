// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello Component should render 1`] = `true`;
